README.md

```diff
@@ -433,7 +433,9 @@ bad
 #### Transformation Signatures
 | Transform | ADT signature | Function Signature |
 |---|---|---|
+| `arrayToList` | `[ a ] -> List a` | `(a -> [ b ]) -> List b` |
 | `eitherToAsync` | `Either e a -> Async e a` | `(a -> Either e b) -> a -> Async e b` |
 | `eitherToMaybe` | `Either b a -> Maybe a` | `(a -> Either c b) -> a -> Maybe b` |
+| `listToArray` | `List a -> [ a ]` | `(a -> List b) -> [ b ]` |
```

Comment:
I was wrong with my initial suggestion. Sorry about that! This should be:

`listToArray` 
`List a -> [ a ]`
`(a -> List b) -> a -> [ b ]`

Forgot to take the `a`. My bad.